### PR TITLE
Poller taking too long -> device took too long

### PR DESCRIPTION
**Pull request by murrant**
We can't really alert on the poller too long since or alerts are device centric.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: misc/alert_rules.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "name": "IPSec tunnels down"
   },
   {
-    "rule": "%pollers.time_taken >= \"250\"",
+    "rule": "devices.last_polled_timetaken >= 290",
```
Comment (laf) [inline misc/alert_rules.json]: Actually shouldn't this be %devices.last_polled_timetaken?

Comment (murrant) [reply misc/alert_rules.json]: We strip out the % before processing anyway. Did you try it?

Comment (laf) [reply misc/alert_rules.json]: No but all of the others have the % in. I'm sure I stripped them out when I did the large alerting PR but seem to recall you added them back in maybe?

Comment (murrant) [reply misc/alert_rules.json]: Might have gotten reverted at some point.

Comment (laf) [reply misc/alert_rules.json]: Also, this changes the alert from the pollers total time to a single device which is fine as the original one obviously won't work but it also needs the name of the rule updating.

**Comment by murrant** [comment]
Removed all % from rules and renamed the poller taking too long rule.

**Comment by laf** [comment]
Thanks for updating these @murrant

**Pull request merged**